CA templates beta public preview (#18315)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/conditionalaccesspolicydetail.md

---
title: "conditionalAccessPolicyDetail resource type"

> description: "Represents details of a Microsoft recommended template for Azure Active Directory conditional access policy."

Reviewer: ```suggestion
description: "Represents the configuration of a Microsoft recommended best-practice template for an Azure Active Directory conditional access policy."
```

Author: fixed

author: "xuchen1"
ms.localizationpriority: medium
ms.prod: "identity-and-sign-in"
doc_type: resourcePageType
---

# conditionalAccessPolicyDetail resource type

Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> Represents details of Conditional Access policy templates.

Reviewer: ```suggestion
Represents the configuration of the **details** property of a Microsoft recommended best-practice [template](../resources/conditionalaccesspolicytemplate.md) for an Azure Active Directory [conditional access policy](../resources/conditionalaccesspolicy.md).
```

Author: fixed!

## Properties
|Property|Type|Description|
|:---|:---|:---|
|conditions|[conditionalAccessConditionSet](../resources/conditionalaccessconditionset.md)| Represents the type of conditions that govern when the policy applies. |

> |grantControls|[conditionalAccessGrantControls](../resources/conditionalaccessgrantcontrols.md)| Represents grant controls that must be fulfilled to pass the policy. |

Reviewer: ```suggestion
|grantControls|[conditionalAccessGrantControls](../resources/conditionalaccessgrantcontrols.md)| Represents grant controls that must be fulfilled for the policy. |
```

Author: fixed
